src/error.rs

```diff
@@ -45,6 +46,7 @@ impl Errors {
             Errors::InvalidOrBlockedHost => {
                 (110, "Invalid or blocked destination host".to_string())
             }
+            Errors::TimedOut => (111, "Connection/Request/Response from the destination timedout".to_string()),
```

Comment:
Do we even need this entire file?

Reply:
maybe src/errors.rs should stay and implement  a Generic errors type for ImageProxy (minus  to_rpc etc )   that  the rpc/error.rs matches on  .. thoughts?

Reply:
hmm lets talk about this on a call.